Client/Phpredis/ClientCluster.php

```diff
@@ -0,0 +1,1545 @@
+<?php
+
+/*
+ * This file is part of the SncRedisBundle package.
+ *
+ * (c) Henrik Westphal <henrik.westphal@gmail.com>
+ * (c) Yassine Khial <yassine.khial@blablacar.com>
+ * (c) Pierre Boudelle <pierre.boudelle@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Snc\RedisBundle\Client\Phpredis;
+
+use RedisCluster;
+use Snc\RedisBundle\Logger\RedisLogger;
+
+/**
```

Comment:
You can remove these docblock entirely

Reply:
I think i didn't understand this suggestion, meantime pushed a new revision